src/components/common/AddressInput/index.tsx

```diff
@@ -29,10 +30,8 @@ const AddressInput = ({ name, validate, required = true, ...props }: AddressInpu
   const isDomainLookupEnabled = !!currentChain && hasFeature(currentChain, FEATURES.DOMAIN_LOOKUP)
   const { address, resolverError, resolving } = useNameResolver(isDomainLookupEnabled ? watchedValue : '')
 
-  // errors[name] doesn't work with nested field names like 'safe.address'.
-  // But getFieldState doesn't trigger a re-render, which breaks tests.
-  // So both are needed to get the error state and pass tests.
-  const fieldError = resolverError || getFieldState(name).error || errors[name]
+  // errors[name] doesn't work with nested field names like 'safe.address', need to use the lodash get
+  const fieldError = resolverError || _get(errors, name)
```

Comment:
This is unrelated, just fixed along the way. Seems to still works with complex field names (tested with owners in Safe creation).

Reply:
RHF exports it's own version of `lodash`' `get` function.